avoidance/src/common.cpp

```diff
@@ -11,10 +11,10 @@ namespace avoidance {
 bool pointInsideFOV(const FOV& fov, const PolarPoint& p_pol) {
   // pitch angle must be counted negative: zero pitch is level, positive pitch
   // is leaning down. Whereas a positive elevation angle is counted upwards
-  return p_pol.z <= fov.yaw_deg + fov.h_fov_deg / 2.f &&
-         p_pol.z >= fov.yaw_deg - fov.h_fov_deg / 2.f &&
-         p_pol.e <= -fov.pitch_deg + fov.v_fov_deg / 2.f &&
-         p_pol.e >= -fov.pitch_deg - fov.v_fov_deg / 2.f;
+  return p_pol.z <= fov.azimuth_deg + fov.h_fov_deg / 2.f &&
+         p_pol.z >= fov.azimuth_deg - fov.h_fov_deg / 2.f &&
+         p_pol.e <= -fov.elevation_deg + fov.v_fov_deg / 2.f &&
+         p_pol.e >= -fov.elevation_deg - fov.v_fov_deg / 2.f;
```

Comment:
I think here the minus is wrong. Now that we use elevation angles, we can use them without the minus